operating/examine-images.adoc

```diff
@@ -0,0 +1,63 @@
+[id="examine-images"]
+= Examining images
+include::modules/common-attributes.adoc[]
+:context: examine-images
+
+toc::[]
+
+[role="_abstract"]
+With {product-title} you can analyze images for vulnerabilities.
+Scanner analyzes all image layers to check for known vulnerabilities by comparing them with the Common Vulnerabilities and Exposures (CVEs) list.
+
+[NOTE]
+====
+To disable language-specific vulnerability scanning run the following command:
+[source,terminal]
+----
+$ oc -n stackrox set env deploy/scanner ROX_LANGUAGE_VULNS=false <1>
+----
+<1> If you are using {product-title} version 3.0.47 or older, replace the environment variable name `ROX_LANGUAGE_VULNS` with `LANGUAGE_VULNS`.
+====
+
+When Scanner finds any vulnerabilities, it:
+
+* Shows them in the xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-view_acs-operating-manage-vulnerabilities[*Vulnerability Management* view] for detailed analysis.
+* Includes them in risk ranking.
```

Comment:
You could either place this item next to the "Highlights them in the portal for risk assessment" or combine them, given that they both relate to risk. For example, you could combine them as follows:

"* Ranks vulnerabilities according to risk and highlights them in the RHACS portal for risk assessment."